ansible/roles/bash_alias/tasks/main.yml

```diff
@@ -8,4 +8,4 @@
       line: "alias {{item.name}}='{{item.command}}'"
     with_items:
       - { name: 'syncmaster', command: 'git checkout master; git fetch --all; git merge --ff-only upstream/master; git push; git checkout -'}
-      - { name: 'pr', command: 'git push origin HEAD -u && hub pull-request'}
+      - { name: 'pr', command: 'hub pull-request -p'}
```

Comment:
I've done this initially first but it did not do what I expected. Have you tested this with forks?

Reply:
aaah no, i haven't. ok will abandon and use it myself